Day-II-Calculator/calculator/src/components/DisplayComponents/CalculatorDisplay.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Display.css';
+
+const CalculatorDisplay = ({result = 0}) => {
```

Comment:
That's cool, I would have never thought of initially assign it as `0` in the parameter.